CHANGELOG.md

```diff
@@ -1,5 +1,278 @@
 # go-ipfs changelog
 
+## 0.4.21 XX-XX-XX
+
+### Commands
+
+#### Fixed: `ipfs add` with multiple files
+
+In version 0.4.20 a bug was introduced preventing `ipfs add fileA fileB` from working.
+This was fixed in ([ipfs/go-ipfs#6255](https://github.com/ipfs/go-ipfs/pull/6255)).
+
+### New: TLS 1.3 handshake
+
+Go-libp2p has gained ability to use TLS 1.3 for connection encryption and authentication.
+Go-ipfs will accept TLS 1.3 handshakes but will still primarily use secio.
+If you want to help testing new handshake, you can enable [experiment](https://github.com/ipfs/go-ipfs/blob/master/docs/experimental-features.md#tls-13-as-default-handshake-protocol).
+
+### New: Build system `GOCC` variable
+
+Build system now uses `GOCC` environment variable allowing for use of multiple
+go versions during builds.
+
+
+(please point out what should be featured, doesn't have to be go-ipfs).
```

Comment:
Should we pull the list of contributors like Alan did in the most recent js-ipfs release? https://github.com/ipfs/js-ipfs/releases/tag/v0.36.0